hs_core/models.py

```diff
@@ -984,9 +975,7 @@ def __unicode__(self):
 
     @classmethod
     def get_supported_types(cls):
-        supported_types = [rel_type for rel_type in dict(cls.SOURCE_TYPES) if rel_type not
-                           in cls.DEPRECATED_RELATION_TYPES]
-        return supported_types
+        return dict(cls.SOURCE_TYPES).keys()
```

Comment:
It seems ```cls.SOURCE_TYPES``` contain deprecated relation types, so it seems this method needs to filter out deprecated types and return the filtered out list, right?

Reply:
@hyi There are no deprecated types. All the types defined for Relation class are now active ones.